packages/appsync-modelgen-plugin/src/interfaces/introspection/model-schema.ts

```diff
@@ -19,6 +19,7 @@
   mutations?: SchemaMutations;
   subscriptions?: SchemaSubscriptions;
   inputs?: SchemaInputs;
+  generations?: SchemaQueries;
```

Comment:
Nit: Consider creating a type alias `GenerationQueries` for `SchemaQueries` and use it for `generations`. It reads little better.

Reply:
Good call. Addressed in [5c412eb](https://github.com/aws-amplify/amplify-codegen/pull/869/commits/5c412eb03f891ae278f54e2834291edcd0646772)